com.unity.render-pipelines.high-definition/Runtime/Material/SubsurfaceScattering/SubsurfaceScattering.compute

```diff
@@ -107,12 +107,12 @@ float4 LoadSampleFromVideoMemory(int2 pixelCoord)
 // Returns {irradiance, linearDepth}.
 float4 LoadSample(int2 pixelCoord, int2 cacheOffset)
 {
+    float4 value;
```

Comment:
just to confirm, you have try different compile option and this wasn't compiling and you ahve fix warning?

Reply:
yes it's for the SSS_USE_LDS_CACHE define